documentation/docs/experimental/index.md

```diff
@@ -24,6 +24,11 @@ The list of experimental features may change as goose development progresses. So
       description="Enable tool calling capabilities for language models that don't natively support tool calling (like DeepSeek) using an experimental local interpreter model setup."
       link="/docs/experimental/ollama"
     />
+    <Card 
+      title="Mobile Access via Secure Tunneling"
+      description="Enable remote access to goose Desktop from the Goose AI mobile app via secure tunneling."
+      link="/docs/experimental/mobile-access"
+    />
     <Card 
       title="goose Mobile"
       description="An experimental Android automation app that acts as an open agent running on your phone, providing maximal automation of everyday tasks."
```

Comment:
i think we can remove this other goose mobile..i think this new version replaces that one.